datasets/conv_ai_2/README.md

```diff
@@ -0,0 +1,183 @@
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://github.com/DeepPavlov/convai/tree/master/2018
+- **Repository:** https://github.com/DeepPavlov/convai/tree/master/2018
+- **Paper:** https://arxiv.org/abs/1902.00098
+- **Leaderboard:** [Needs More Information]
+- **Point of Contact:** [Needs More Information]
+
+### Dataset Summary
+
+ConvAI is a dataset of human-to-bot conversations labeled for quality. This data can be used to train a metric for evaluating dialogue systems. Moreover, it can be used in the development of chatbots themselves: it contains information on the quality of utterances and entire dialogues, that can guide a dialogue system in search of better answers.
+
+### Supported Tasks and Leaderboards
+
+[Needs More Information]
+
+### Languages
+
+[Needs More Information]
+
+## Dataset Structure
+
+### Data Instances
+
+{
+        "dialog_id": "0x648cc5b7",
+        "dialog": [
+            {
+                "id": 0,
+                "sender": "participant2",
+                "text": "Hi! How is your day? \ud83d\ude09",
+                "sender_class": "Bot"
+            },
+            {
+                "id": 1,
+                "sender": "participant1",
+                "text": "Hi! Great!",
+                "sender_class": "Human"
+            },
+            {
+                "id": 2,
+                "sender": "participant2",
+                "text": "I am good thanks for asking are you currently in high school?",
+                "sender_class": "Bot"
+            }
+        ],
+        "bot_profile": [
+            "my current goal is to run a k.",
+            "when i grow up i want to be a physical therapist.",
+            "i'm currently in high school.",
+            "i make straight as in school.",
+            "i won homecoming queen this year."
+        ],
+        "user_profile": [
+            "my favorite color is red.",
+            "i enjoy listening to classical music.",
+            "i'm a christian.",
+            "i can drive a tractor."
+        ],
+        "eval_score": 4,
+        "profile_match": 1
+    }
```

Comment:
````suggestion
```
{
        "dialog_id": "0x648cc5b7",
        "dialog": [
            {
                "id": 0,
                "sender": "participant2",
                "text": "Hi! How is your day? \ud83d\ude09",
                "sender_class": "Bot"
            },
            {
                "id": 1,
                "sender": "participant1",
                "text": "Hi! Great!",
                "sender_class": "Human"
            },
            {
                "id": 2,
                "sender": "participant2",
                "text": "I am good thanks for asking are you currently in high school?",
                "sender_class": "Bot"
            }
        ],
        "bot_profile": [
            "my current goal is to run a k.",
            "when i grow up i want to be a physical therapist.",
            "i'm currently in high school.",
            "i make straight as in school.",
            "i won homecoming queen this year."
        ],
        "user_profile": [
            "my favorite color is red.",
            "i enjoy listening to classical music.",
            "i'm a christian.",
            "i can drive a tractor."
        ],
        "eval_score": 4,
        "profile_match": 1
    }
```
````